lib/chai/utils/overwriteProperty.js

```diff
@@ -4,7 +4,9 @@
  * MIT Licensed
  */
 
+var chai = require('../../chai');
```

Comment:
I'm a bit confused how it's working.

1. `test/bootstrap/index.js` requires `index.js`
1. `index.js` requires `lib/chai.js`
1. `lib/chai.js` requires `lib/chai/utils/index.js`
1. `lib/chai/utils/index.js` requires `lib/chai/utils/addProperty.js`
1. `lib/chai/utils/addProperty.js` requires `lib/chai.js`

Thus circular dependency. -scratches head-